hyde-themes.db

```diff
@@ -86,3 +86,23 @@ DESCRIPTION:""
 LINK="https://github.com/prasanthrangan/hyde-themes/tree/Rose-Pine"
 OWNER="https://github.com/prasanthrangan"
 DESCRIPTION:""
+
+[AI Arts]
+LINK="https://github.com/tryprncp/Themes/tree/AI-Arts"
+OWNER="https://github.com/tryprncp"
+DESCRIPTION:""
+
+[Abstract]
+LINK="https://github.com/tryprncp/Themes/tree/Abstract"
+OWNER="https://github.com/tryprncp"
+DESCRIPTION:""
+
+[E-girls]
```

Comment:
![image](https://github.com/kRHYME7/hyde-gallery/assets/53417443/e3dc8e45-67cd-47c4-885e-2171ec2a85fe)

Colors do not fit the theme i guess we need to get some Ideas on other Themes.